specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2896,18 +2896,18 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/triggers/{triggerName}/rerunTriggers/{rerunTriggerName}": {
-      "put": {
```

Comment:
possible break changes here. Consider use a new api version for these changes?

Reply:
This feature/api has been blocked since past 3 months, since this api was not supposed to be released. The original routes( for all rerunTriggers tags) are to be removed, this is the new api which we are exposing for the feature, merging the functionality with the existing resource, instead of creating a new one.

Reply:
So the current api version is not exposed yet right?

Reply:
This is an important question. If it has been exposed in a stable API version, we have to depricate it, not just delete it. If swagger was updated (by mistake) and the service never exposed it, we can just clean it up by deleting it.